ci/run-tests.ps1

```diff
@@ -57,11 +58,20 @@ $cmd_args_list = @( `
     "-nopause", ` # Close the unreal log window automatically on exit
     "-nosplash", ` # No splash screen
     "-unattended", ` # Disable anything requiring user feedback
-    "-nullRHI", # Hard to find documentation for, but seems to indicate that we want something akin to a headless (i.e. no UI / windowing) editor
-    "-OverrideSpatialNetworking=`"$override_spatial_networking`"" # A parameter to switch beetween different networking implementations
+    "-nullRHI" # Hard to find documentation for, but seems to indicate that we want something akin to a headless (i.e. no UI / windowing) editor
 )
 
+if($run_with_spatial) {
+    $cmd_args_list += "-OverrideSpatialNetworking=$run_with_spatial" # A parameter to switch beetween different networking implementations
+}
+
 Echo "Running $($ue_path_absolute) $($cmd_args_list)"
 
 $run_tests_proc = Start-Process $ue_path_absolute -PassThru -NoNewWindow -ArgumentList $cmd_args_list
-Wait-Process -Id (Get-Process -InputObject $run_tests_proc).id
+try {
+    Wait-Process -Timeout 1800 -InputObject $run_tests_proc
```

Comment:
What is this Timeout value?

Reply:
30 minutes for a test run, before it is forcefully killed to avoid builds which hang and never upload logs.

Reply:
Add comment